lib/Migration/FixVotes.php

```diff
@@ -72,20 +65,19 @@ public function getName() {
 	 * @return void
 	 */
 	public function run(IOutput $output) {
-		$schema = new SchemaWrapper($this->connection);
-		if ($schema->hasTable('polls_options')) {
-			$table = $schema->getTable('polls_options');
-			if (!$table->hasColumn('duration')) {
-				$foundOptions = $this->optionMapper->findOptionsWithDuration();
-				foreach ($foundOptions as $option) {
-					$this->voteMapper->fixVoteOptionText(
-						$option->getPollId(),
-						$option->getId(),
-						$option->getPollOptionTextStart(),
-						$option->getPollOptionText()
-					);
-				}
+		try {
+			$foundOptions = $this->optionMapper->findOptionsWithDuration();
+			\OC::$server->getLogger()->error(json_encode($foundOptions));
```

Comment:
this will now log an error for everyone, even for successful runs

Reply:
😆 Just removed it. Thanks. 👍